noxfile.py

```diff
@@ -0,0 +1,22 @@
+"""Nox sessions for Jupyter Book documentation."""
+
+import nox
+
+# Use uv for faster installs
+nox.options.default_venv_backend = "uv|virtualenv"
+
+
+@nox.session(name="docs")
+def docs(session):
+    """Build the documentation as static HTML."""
+    session.install("-e", ".[docs]")
+    session.chdir("docs")
+    session.run("jupyter", "book", "build", "--html", "--execute")
```

Comment:
maybe you want to run this with `--strict` so the CI would pick up on missing pages/links?